velox/docs/monitoring/stats.rst

```diff
@@ -39,3 +40,26 @@ Memory Arbitration
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+
+HashBuild, HashAggregation
+--------------------------
+
```

Comment:
Let's clarify somehow that these stats are reported only by HashBuild and HashAggregation operators.